dsc/config.py

```diff
@@ -10,9 +10,9 @@ class Config:
         "WORKSPACE",
         "SENTRY_DSN",
         "AWS_REGION_NAME",
-        "DSC_SOURCE_EMAIL",
-        "DSS_INPUT_QUEUE",
-        "S3_BUCKET",
+        "S3_BUCKET_SUBMISSION_ASSETS",
+        "SOURCE_EMAIL",
+        "SQS_QUEUE_DSS_INPUT",
```

Comment:
Optional: I would do this as `SQS_DSS_INPUT_QUEUE` but your call!

Reply:
I'll probably keep as-is for now, to line up with what we listed in our discussion!